elements/pfe-sass/mixins/_copy-mixins.scss

```diff
@@ -173,3 +173,94 @@
   text-decoration: underline;
   text-decoration-skip: ink;
 }
+
+
+/// HTML TYPOGRAPHY
+
+/// BASE LEVEL TYPOGRAPHY
+/// for the body tag
+@mixin pfe-text-base {
+    font-family: pfe-var(font-family);
+    font-weight: pfe-var(font-weight--light);
+    line-height: pfe-var(line-height);
+}
+
+/// TEXT MODIFIER CLASSES
+@mixin pfe-text ( $sizing: "md", $base: true) {
+  // these styles are ordinarily inherited from the body tag, but some projects may need extra specificity
+  @if $base == true {
+      @include pfe-text-base;
```

Comment:
This seems like the perfect place for the extend.  If you're using the text mixin alone, you get the styles from the extend no issue.  If you use this mixin to print out multiple text modifiers, yay you get the added efficiency on render of the base pulling the shared styles together!